views/layouts/admin_main.handlebars

```diff
@@ -164,16 +178,17 @@
     <!-- ============================================================== -->
     <!-- End Wrapper -->
     <!-- ============================================================== -->
-   
+
 </body>
 {{!-- JavaScript --}}
-  <script>
-        let date = "{{viewData.rhapsody.date}}";
-        let newDate = new Date(date);
-        let DD = newDate.getDate() < 10 ? '0' + newDate.getDate() : newDate.getDate();
-        let MM = newDate.getMonth() < 10 ? '0' + newDate.getMonth() : newDate.getMonth();
-        let YYYY = newDate.getFullYear();
-        let YYYYMMDD = YYYY + '-' + MM + '-' + DD;
-        document.getElementById("rhapsodyDate").value = YYYYMMDD;
-    </script>
+<script>
+    let date = "{{viewData.rhapsody.date}}";
+    let newDate = new Date(date);
+    let DD = newDate.getDate() < 10 ? '0' + newDate.getDate() : newDate.getDate();
+    let MM = (newDate.getMonth() + 1) < 10 ? '0' + (newDate.getMonth() + 1) : (newDate.getMonth() + 1);//Adds one because months starts at 0 not 1
+    let YYYY = newDate.getFullYear();
+    let YYYYMMDD = YYYY + '-' + MM + '-' + DD;
+    document.getElementById("rhapsodyDate").value = YYYYMMDD;
```

Comment:
don't really know what we need this frontend logic for, but i guess you do, when i have time latter, i'll check, anywas that does not block this pr